src/services/language.js

```diff
@@ -0,0 +1,162 @@
+import importedLibs from 'lang';
+import Kleio from 'kleio';
+import store from 'flux/store';
+import deepExtend from 'deep-extend';
+
+/**
+ * Service that manages imported language libs. Based on language code
+ */
+class LanguageService {
+  constructor(language = LanguageService.defaultLanguage) {
+    const kleio = new Kleio(null, Kleio.ENV_MODES.DEV);
+
+    this._record = kleio.record.bind(kleio);
+
+    this._listeners = [];
+
+    this.language = language;
+
+    this._setLanguageFromStore();
+    store.subscribe(this._setLanguageFromStore);
+  }
+
+  static defaultLanguage = 'en';
```

Comment:
Place static variables outside the class body, this will cause JSDoc generator to fail atm... 
```javascript
LanguageService.defaultLanguage = 'en'
```